test/integrations/destinations/am/processor/data.ts

```diff
@@ -791,6 +796,9 @@ export const data = [
                       insert_id: '84e26acc-56a5-4835-8233-591137fca468',
                       ip: '0.0.0.0',
                       user_properties: {
+                        $unset: {
+                          'testObj.unsetField1': '-',
```

Comment:
Does this reflect correctly over at amplitude, like remove the nested property inside testObj?

Reply:
Yes thats already verified